src/components/image/CdrImg.vue

```diff
@@ -106,8 +105,8 @@ export default defineComponent({
     const baseClass = 'cdr-image';
     const ratioClass = 'cdr-image-ratio';
     const coverWrapperClass = 'cdr-image-ratio__cover';
-    const modifierClass = computed(() => buildClass(baseClass, props.modifier));
-    const radiusClass = computed(() => buildClass(baseClass, props.radius));
+    const modifierClass = computed(() => props.modifer && `${baseClass}--${props.modifer}`);
+    const radiusClass = computed(() => props.radius && `${baseClass}--${props.radius}`);
```

Comment:
You'll notice a lot more null-checking at the component level (e.g. `props.radius && ...`) and that's because it was previously being done in the buildClass method.